Tests/StreamChatUITests/ChatMessage_Tests.swift

```diff
@@ -68,4 +70,311 @@ final class ChatMessage_Tests: XCTestCase {
         
         XCTAssertEqual(message.lastActiveThreadParticipant?.name, "Second")
     }
+
+    // MARK: - isInteractionEnabled
+
+    func test_isInteractionEnabled_returnsFalse_forEphemeralMessage() {
```

Comment:
```suggestion
    func test_isInteractionEnabled_whenEphemeralMessage_returnsFalse() {
```
We should write a doc for the naming test convention. But usually, the expected behaviour comes last. We should try to follow the pattern: 

##### `test_Given_When_Then`
Given -> Name of the func/prop
When -> Condition
Then -> Expected result

This is also true for the rest of the test cases in ChatMessage_Tests

Reply:
give me 5 min 😄

Reply:
✅ Done